release-notes/aws/v9.0.1.md

```diff
@@ -0,0 +1,25 @@
+## :zap: Giant Swarm Release 9.0.1 for AWS is now active for you! :zap:
+
+This release includes some reliability and user experience improvements.
+
+Below, you can find more details on components that were changed with this release.
+
+### chart-operator [v0.12.0](https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0120-2020-03-09)
+
+- Added chartmigration resource for migrating from chartconfig to chart CRs.
+- Updated release resource.
+  - Do not wait when installing or updating long running Helm releases.
+  - Use version field from chart CR to reduce number of HTTP requests to pull chart tarballs.
+  - Wait for deleted Helm release before removing finalizer. 
+- Updated status resource.
+  - Improve reason field in CR status when installing a chart fails.
+- Removed legacy chartconfig controller.
+
+### cluster-operator [v0.23.4](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.4)
+
+- Stop setting IC replicas count.
```

Comment:
Since 9.0.0 was using cluster-operator 0.21.0, I collected all the logs between `0.23.4..0.21.0`.

Reply:
FYI for this release notes there are still TODOs
- I didn't update cluster-operator release notes section, while I did change release definition to use cluster-operator 0.23.5 instead of 0.23.4.

Reply:
Covered everything up to cluster-operator 0.23.8